README.md

```diff
@@ -23,23 +23,27 @@ More details can be found on the [IDR website](https://idr.openmicroscopy.org/ab
 BaSiCPy is the python implementation of the BaSiC method for illumination correction, introduced in [Peng, T et al., 2020](https://doi.org/10.1007/978-3-030-59722-1_17).
 - [CellPose](https://github.com/mouseland/cellpose) is used to segment nuclei from mitosis movies.
 CellPose was first introduced in [Stringer, C., Wang, T., Michaelos, M. et al., 2020](https://doi.org/10.1038/s41592-020-01018-x).
-- [DeepProfiler](https://github.com/cytomining/DeepProfiler) is used to extract features from mitosis movies. 
+- [DeepProfiler](https://github.com/cytomining/DeepProfiler) is an option to extract deep learning features from mitosis movies. 
+- [CellProfiler](https://github.com/CellProfiler/CellProfiler) is an option for segmentation and feature extraction from mitosis movies. 
```

Comment:
```suggestion
- [CellProfiler](https://github.com/CellProfiler/CellProfiler) can be used for segmentation and traditional feature extraction from mitosis movies. 
```

Not sure the "traditional" is necessary but want to emphasize the difference between DP and CP

Reply:
:+1: